examples/connectivity/plot_extract_regions_canica_maps.py

```diff
@@ -0,0 +1,139 @@
+"""
+Regions extraction using Canonical ICA maps and functional connectomes
+======================================================================
+
+This example shows how to use :class:`nilearn.regions.RegionExtractor`
+to extract connected brain regions from whole brain ICA maps and
+use them to estimate a connectome.
+
+We used 20 resting state ADHD functional datasets from :func:`nilearn.datasets.fetch_adhd`
+and :class:`nilearn.decomposition.CanICA` for whole brain ICA maps.
+
+Please see the related documentation of :class:`nilearn.regions.RegionExtractor`
+for more details.
+"""
+
+################################################################################
+# Fetching ADHD resting state functional datasets by loading from datasets
+# utilities
+from nilearn import datasets
+
+adhd_dataset = datasets.fetch_adhd(n_subjects=20)
+func_filenames = adhd_dataset.func
+confounds = adhd_dataset.confounds
+
+################################################################################
+# Canonical ICA decomposition of functional datasets by importing CanICA from
+# decomposition module
+from nilearn.decomposition import CanICA
+
+# Initialize canica parameters
+n_components = 5
```

Comment:
Can you remove the n_components variable, and simply pass the value "5" below, in an effort to simplify things for users.